src/shared/components/event-start/services/event-start-state-machine.service.ts

```diff
@@ -197,9 +197,8 @@ export class EventStartStateMachineService {
   }
 
   public finalAction(state: State): void {
```

Comment:
is this function even needed if it only does console.log or does it just need to be stubbed out for now?

Reply:
I'll comment out the log for now - think it was left in as precursor to other potential requirements